assembl/tests/views_tests/test_graphql.py

```diff
@@ -2495,3 +2495,244 @@ def test_query_discussion_homepage(graphql_request, discussion, test_session):
 """
     res = schema.execute(query, context_value=graphql_request)
     assert res.data['discussion']['homepageUrl'] == url
+
+
+def test_query_discussion_langstring_fields(discussion, graphql_request):
+    res = schema.execute(u"""query {
+        discussion {
+            title
+            titleEntries {
+                value
+                localeCode
+            }
+            subtitle
+            subtitleEntries {
+                value
+                localeCode
+            }
+            buttonLabel
+            buttonLabelEntries {
+                value
+                localeCode
+            }
+        }
+    }""", context_value=graphql_request)
+    assert res.errors is None
+    res_data = json.loads(json.dumps(res.data))
```

Comment:
why!?

Reply:
Ah I thought because `res.data` uses OrderedDict the assert wouldn't work, but it actually works.

Reply:
ok !